src/Symfony/Component/Security/.gitattributes

```diff
@@ -0,0 +1,5 @@
+Core/Tests export-ignore
+Csrf/Tests export-ignore
+Guard/Tests export-ignore
+Http/Tests export-ignore
```

Comment:
isn't this list the same as just one line, which applies recursively: `Tests export-ignore`?

Reply:
The file can be removed as the `symfony/security` component is deprecated/obsolete.

Reply:
That is true. `.gitattributes` is using basically the same format as `.gitignore` ([docs](https://git-scm.com/docs/gitignore#_pattern_format)). 

I've removed this file.